template.yaml

```diff
@@ -806,6 +806,30 @@ Resources:
             Method: get
             RestApiId: !Ref NvaPublicationApi
 
+  NvaFetchImportCandidateFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: publication-rest
+      Handler: no.unit.nva.publication.fetch.FetchImportCandidateHandler::handleRequest
+      Runtime: java11
+      AutoPublishAlias: live
+      DeploymentPreference:
+        Type: AllAtOnce
+      ProvisionedConcurrencyConfig:
+        ProvisionedConcurrentExecutions: 1
```

Comment:
Do we want to pay extra for concurrency yet?
(we will in future, but right now it will be some time before users will use this endpoint).